### ISSUE TITLE
User/Application guide

### ISSUE THREAD

**Issue by DaJansenGit** [BIM2SIM/bim2sim#595]

**Comment by Cudok** [comment]
First Steps:

@DaJansenGit  1. is missing in the actual version of the new structure?

**Comment by Cudok** [comment]
Plugins:

1. The whole installation (including base/core bim2sim) should be part of the installation guide of the specific plugin?
  - I think: only the plugin specific part should be in the plugin part 
  - but a link to the base/core installation should be included.

**Comment by Cudok** [comment]
General Question: 
1. Is Bim2Sim a Framework ?
  - or a Library?
  -  How do we want to call it in the documentation?

**Comment by DaJansenGit** [comment]
> General Question:
> 
> 1. Is Bim2Sim a framework ?
> 
> * or a Library?
> * How do we want to call it in the documentation?
> * I use framework, for now. I can change it later, if needed.
> 
> 2. syle of bim2sim, always `bim2sim` (is always highlighted)
> 
> * i will do it for now. (change it if needed)

I agree to both. `bim2sim` is a framework. In our paper, we are currently working on, we call `bim2sim` a framework as well.
